File: Python/level 2/2016.py
def getDayName(a,b):
    mon = [31,29,31,30,31,30,31,31,30,31,30,31]
    day = ["FRI","SAT","SUN","MON","TUE","WED","THU"]
    return day[((sum(mon[0:a-1])+b)%7)-1]

#아래 코드는 테스트를 위한 출력 코드입니다.
print(getDayName(5,24))

File: CPP/level 2/2016.cpp
#include<iostream>
#include<string>
using namespace std;

string getDayName(int a,int b)
{
	int mon[12] = {31,29,31,30,31,30,31,31,30,31,30,31};
  string day[7]={"FRI","SAT","SUN","MON","TUE","WED","THU"};
  string answer= "";
	int sum=0;
  
  for(int i=0; i<a-1; i++){
    sum+=mon[i];
  }
  sum += b;
  answer=day[(sum%7)-1];
	return answer;
}
int main()
{
	int a=5,b=24;

	//아래는 테스트 출력을 위한 코드입니다.
	cout<<getDayName(a,b);
}
